test/datomish/test/query.cljc

```diff
@@ -735,6 +735,34 @@
   [a b]
   (= (set a) (set b)))
 
+(deftest-db test-find-specs-empty-results conn
```

Comment:
Kinda think we need a `deftest-schema` that opens the conn and transacts the initial schema.  Sometime later.